The-SMIRNOFF-force-field-format.md

```diff
@@ -663,79 +675,5 @@ This is a backwards-incompatible overhaul of the SMIRNOFF 0.1 draft specificatio
 
 Initial draft specification.
 
-## Open Questions
```

Comment:
Instead of deleting all these wholesale, we should migrate these questions somewhere: Perhaps an issue for now?

Reply:
We've had this Issue for stray spec questions open for a while, so I had added them here. https://github.com/openforcefield/openforcefield/issues/120#issuecomment-480447124

Reply:
Great, thanks!